sdk/python/tests/compiler/testdata/artifact_passing_using_volume.yaml

```diff
@@ -115,7 +115,7 @@ spec:
                 pass
             with open(output_file, 'w') as f:
                 f.write(_output_serializers[idx](_outputs[idx]))
-      image: python:3.7
+      image: python:3.9
```

Comment:
These tests under `sdk/python/tests/` are associated with KFP SDK v1, so let's not bother to update them. This is undocumented knowledge, apologies. We're moving toward cleaning this up.